functions_framework/lib/functions_framework.dart

```diff
@@ -12,6 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// Provides the features needed to *define* Cloud Functions.
+///
+/// ```dart
+/// // lib/functions.dart
+/// import 'package:functions_framework/functions_framework.dart';
+/// import 'package:shelf/shelf.dart';
+///
+/// @CloudFunction()
+/// Response function(Request request) => Response.ok('Hello, World!');
+// ```
```

Comment:
Can we use // or /// consistently in this comment? The \`\`\` indentation does not align.

Reply:
Yeah, I think it's just the last line of the doc comment that needs to be `///`.

Reply:
that's a mistake on my part!